### PR TITLE
Feature/session view

### PR DESCRIPTION
**Pull request by CanerShefik**

### CODE REVIEW

File: src/main/components/SessionView.js [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+                        <TalkCard talk={talkDetail}></TalkCard>
+                    </Col>
+                </Row>
+
+                <WhiteText>Speakers: </WhiteText>
+
+                {/*Imports for the Speaker card are within the imports, the parameters are then provide for them to be displayed*/}
+                <Row around="xs">
+                    <SpeakerCard name="Test Speaker" company="Capgemini" blog="personalblog.com" talks={["Intro to Devoxx (Room 1 - 11:45)"]} ></SpeakerCard>
+                    <SpeakerCard name="Test Speaker" company="Capgemini" blog="personalblog.com" talks={["Intro to Devoxx (Room 1 - 11:45)"]}></SpeakerCard>
```
Comment (daniel-cotton) [inline src/main/components/SessionView.js]: these should be a part of the talkDetail object (which in itself should also be passed in from the 'report' page)

File: src/main/components/SpeakerCard.js [hunk 2]
```diff
@@ -18,14 +27,30 @@ const ImgCircle = styled.img`
...
 `;
 
+const Container = styled.div`
+    width: 70%;
+    paddingBottom: 10px;
+    text-align: left;
+    margin-left : 20px;
+    
+    
+`;
```
Comment (daniel-cotton) [inline src/main/components/SpeakerCard.js]: need any more line breaks?

File: src/main/components/Report.js [hunk 3]
```diff
@@ -1,64 +1,75 @@
...
+    'volutpat qualisque. Sale tantas cotidieque quo ut, ad nostro consectetuer' +
+    ' nec. Feugiat qualisque quo an. Labores officiis te nam.',
+    review: {
+        name: 'Test User',
+        comment: 'Great session, thanks for organising. Looking forward to the next one!',
+        image: testImage
+    }
 };
 
+const speakerDetail = {
```
Comment (daniel-cotton) [inline src/main/components/Report.js]: Shouldn't be here?! @sam-eade - all data should be at 'App.js' level and passed down via props

File: src/main/components/Report.js [hunk 4]
```diff
@@ -1,64 +1,75 @@
...
+        company: 'Capgemini',
+        blog: 'personalblog.com',
+        talks: [
+            'Intro to Devoxx (Room 1 - 11:45)',
+            'Intro to Devoxx 2 (Room 2 - 13:45)'
+        ]
+};
 
-let reportStatsData =
+const reportStatsData =
```
Comment (daniel-cotton) [inline src/main/components/Report.js]: As Above.

File: src/main/components/Report.js [hunk 5]
```diff
@@ -1,64 +1,75 @@
...
-  }
+        return (
+            <div className="Report">
+                <Row center="xs">
+                    <Col xs={10}>
+                        <ReportStats minutes={minutes} talks={talks} learning={learning}
+                                     attendees={attendees}/>
+                    </Col>
+                </Row>
+                <div style={{height: '72px'}}>
```
Comment (daniel-cotton) [inline src/main/components/Report.js]: Empty div? why's this here??

File: src/main/components/SessionView.js [hunk 6]
```diff
@@ -0,0 +1,44 @@
...
+        return (
+            <Window>
+
+                <DayText>Day {this.props.dayNo}</DayText>
+
+                <WhiteText>{this.props.sTime} - {this.props.room}</WhiteText>
+
+                <Row center="xs">
+                    <Col xs={10}>
+                        <Report>{this.props.report}</Report>
```
Comment (daniel-cotton) [inline src/main/components/SessionView.js]: Session view shouldn't contain a report, should be the other way round. Report contains session views which contain session cards .etc

File: src/main/components/SpeakerCard.js [hunk 7]
```diff
@@ -19,46 +24,60 @@ const ImgCircle = styled.img`
...
+                            <div>Personal Blog:</div>
+                            <Blog>{blog}</Blog>
+                        </Col>
+                    </Row>
 
+                    <Row>
+                        <Col>
+                            <div>Other Talks:</div>
+                            <ul>
+                                <Blog>
```
Comment (daniel-cotton) [inline src/main/components/SpeakerCard.js]: Why are other talks in a Blog tag?

File: src/test/snapshot/App.test.js [hunk 8]
```diff
@@ -3,7 +3,17 @@ import ReactDOM from 'react-dom';
 import App from '../../main/App';
 import renderer from 'react-test-renderer';
 
-test('App component', () => {
+
+let reportStatsData =
+    {
+        "minutes": 455,
+        "talks": 10,
+        "learning": "JS, Polymer, Java" ,
```
Comment (daniel-cotton) [inline src/test/snapshot/App.test.js]: app data for snapshot should be a complete dataset of all info in the app. Fill in all props not just report stats

File: src/main/components/Report.js (hunk 5 above)
Comment (sam-eade) [inline src/main/components/Report.js]: no idea why that was there so i removed it

**Pull request merged**